[2022-08-12 11:09:33,307] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-12T11:09:30.533518+00:00 [queued]>
[2022-08-12 11:09:33,319] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-12T11:09:30.533518+00:00 [queued]>
[2022-08-12 11:09:33,319] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-12 11:09:33,320] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-12 11:09:33,320] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-12 11:09:33,338] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): download_raw_data> on 2022-08-12 11:09:30.533518+00:00
[2022-08-12 11:09:33,343] {standard_task_runner.py:52} INFO - Started process 556 to run task
[2022-08-12 11:09:33,348] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_Raízen', 'download_raw_data', 'manual__2022-08-12T11:09:30.533518+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpjrznfvvz', '--error-file', '/tmp/tmpb7nnw44y']
[2022-08-12 11:09:33,350] {standard_task_runner.py:80} INFO - Job 134: Subtask download_raw_data
[2022-08-12 11:09:33,421] {task_command.py:369} INFO - Running <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-12T11:09:30.533518+00:00 [running]> on host a471224fadbb
[2022-08-12 11:09:33,512] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_Raízen
AIRFLOW_CTX_TASK_ID=download_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-12T11:09:30.533518+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-12T11:09:30.533518+00:00
[2022-08-12 11:09:33,514] {ETL_Raizen.py:41} INFO - Downloading .xls raw data
[2022-08-12 11:09:37,472] {ETL_Raizen.py:43} INFO - Succesfully downloaded .xls raw data
[2022-08-12 11:09:37,474] {python.py:173} INFO - Done. Returned value was: None
[2022-08-12 11:09:37,518] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Raízen, task_id=download_raw_data, execution_date=20220812T110930, start_date=20220812T110933, end_date=20220812T110937
[2022-08-12 11:09:37,589] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-12 11:09:37,719] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
